doc/charm++/manual.rst

```diff
@@ -8342,7 +8370,9 @@ Section :numref:`loadbalancing`) after restart is suggested.
 If restart is not done on the same number of processors, the
 processor-specific data in a group/nodegroup branch cannot (and usually
 should not) be restored individually. A copy from processor 0 will be
-propagated to all the processors.
+propagated to all the processors. Additionally, singleton chares will
+not be restored at all in this case, so they must be specially handled
+when restarting on a different number of processors.
```

Comment:
Thanks for noting this behavior! Sanity check on my understanding: I interpret this to mean that some unpacking pup (probably some mainchare) needs to call something like `if (new number of procs) { my_singleton_proxy_ = CProxy_MySingleton::ckNew(args) }` .. is this about right? Adding a note to this effect (or fixing my understanding if needed) as an example might be helpful to close out this paragraph.

Reply:
Indeed, something of that ilk would work. I've added a sentence to clarify this.